### PR TITLE
IPv4 verifier

### PR DESCRIPTION
**Pull request by Hruuttila**
## Status
- [] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Script to verify if the input or list of inputs is IPv4 address.

## Screenshots

Screenshot 2022-02-16 at 16 31 57

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by bakatzir** [comment]
For the reviewer, as this is a well needed transformer, as https://github.com/demisto/content/tree/0d6b34946480c99330b4a957bbe0a408748b6508/Packs/CommonScripts/Scripts/VerifyIPv6Indicator, I suggest adding it to `CommonScripts`. Eng Call. @yuvalbenshalom FYI.

File: Packs/VerifyIPv4Indicator/Scripts/VerifyIPv4Indicator.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+script: ''
+type: python
+tags:
+- indicator-format
+comment: Verify that the address is a valid IPv4 address.
+enabled: true
+args:
+- name: input
+  default: true
+  description: The IPv6 to vriefy.
```
Comment (bakatzir) [inline Packs/VerifyIPv4Indicator/Scripts/VerifyIPv4Indicator.yml:15]:
```suggestion
  description: The IPv4 to verify.
```

File: Packs/VerifyIPv4Indicator/Scripts/VerifyIPv4Indicator.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+commonfields:
+  id: b2d200b7-9f6f-43e8-84c9-07d1084739f1
+  version: 5
+vcShouldKeepItemLegacyProdMachine: false
```
Comment (bakatzir) [inline Packs/VerifyIPv4Indicator/Scripts/VerifyIPv4Indicator.yml:4]:
```suggestion
  id: VerifyIPv4Indicator
  version: -1
```

**Comment by yuvalbenshalom** [comment]
@bakatzir I agree. this should be over to commonScripts.

**Comment by yuvalbenshalom** [comment]
dismiss my last comment. checking if this should be moved or not.

**Comment by yuvalbenshalom** [comment]
@Hruuttila @bziser @bakatzir after some checking, let's keep this in its own pack. we do not want to add more scripts to common scripts.

**Comment by altmannyarden** [comment]
Hi, Is the script also check if the ip4v input is a valid IP? (for example `8.8.8.343` is not a valid IP)

**Comment by Hruuttila** [comment]
@altmannyarden yes. If its not a valid IPv4 it does not return anything. If its valid it returns the IP.
Screenshot 2022-02-18 at 7 55 27

**Review by bziser — APPROVED** [review]

**Pull request merged**